src/autoload/MultiplayerInfo.gd

```diff
@@ -30,6 +30,11 @@ func _player_disconnected(id):
 	print("Player id %d disconnected" % [id])
 	# warning-ignore:return_value_discarded
 	player_info.erase(id) # Erase player from info.
+	# Call function to update lobby UI here
+	var game = get_tree().get_root().get_node_or_null("Game") as Node
```

Comment:
pedantic, use `:=` instead of `=` (so that the whole `as Node` is actually doing something)